--- old/linux-gpib-user/language/tcl/ibCmds.c	2024-12-11 12:29:21.000000000 -0500
+++ new/linux-gpib-user/language/tcl/ibCmds.c	2025-03-26 09:35:54.687472516 -0400
@@ -1,11 +1,12 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
-
-
 #include <tcl.h>
 #include <gpib/ib.h>
 
+#   undef _ANSI_ARGS_
+#   define _ANSI_ARGS_(x)       x
+
 void ib_CreateVerboseError(Tcl_Interp *interp,char *entry );
 
 int Gpib_tcl_Init ( Tcl_Interp *interp ){
